### PR TITLE
Modified install docs to account for bundler

### PR DESCRIPTION
**Pull request by Ommy**
Currently doing bundle install inside chef requires 1.9.3 Ruby.
Also, bundler is a gem, so we need to get gem and then do a `gem install bundler`

### CODE REVIEW

**Comment by Trinovantes** [comment]
We should just break up `INSTALL.md` into two files:
- `SETUP_DEVELOPMENT.md`
- `DEPLOY.md`

**Comment by ConnorCimowsky** [comment]
LGTM

File: docs/INSTALL.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ This will vary depending on your OS, but here's an example using Ubuntu:
 
 ```shell
 apt-get update
-apt-get install git bundler python-pip
+apt-get install git gem python-pip
```
Comment (Kapin) [inline docs/INSTALL.md]: Doesn't ruby come with RubyGems?

**Comment by Kapin** [comment]
One question otherwise :+1:

File: docs/INSTALL.md (hunk 1 above)
Comment (Ommy) [inline docs/INSTALL.md]: Ah you're right, Ruby 1.9+ comes with RubyGems installed. I'll remove it.

**Comment by Trinovantes** [comment]
+1

**Pull request merged**